packages/esbuild/package.json

```diff
@@ -34,7 +34,6 @@
     "@nrwl/workspace": "file:../workspace",
     "chalk": "4.1.0",
     "dotenv": "~10.0.0",
-    "esbuild": "^0.15.7",
```

Comment:
@jaysoo Hey Jack, Sorry I'm just curious why isn't `esbuild` a peer dependency?

Reply:
Yeah i can add it as optional peer dep. The esbuild generator will add it for you if not already installed.